prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts

```diff
@@ -42,8 +32,6 @@ export function getTipsTipCall(
       if (call.isV28) {
         return call.asV28;
       }
-
-      return call.asLatest;
     }
 
```

Comment:
I think we either should have an exception in the place of the return call.asLatest or we should change the fallback exception message.

Reply:
We do have it, it's the `default`. Do you think I should make the code more readable?

<img width="669" alt="Screenshot 2022-08-16 at 09 01 21" src="https://user-images.githubusercontent.com/2465005/184828788-454fffb9-89d5-43eb-86f0-36972b02c44e.png">

Reply:
Or the error more explicit :D

Reply:
Sorry that is what I meant by fallback exception message (bad wording on my part). That exception should mean we don't have a type getter for that network. We should either make it more generic... or probably preferably just add an 'unexpected version' error. Should we create a new error type that we use for just this purpose everywhere? Then we can alert on it in future?